konnector.js

```diff
@@ -85,17 +86,19 @@ function login(requiredFields, billInfos, data, next) {
         logger.info('Fetching bills list')
         request(billsOptions, (err, res, body) => {
             if (err) {
+                logger.error('LOGIN_FAILED');
                 logger.error('An error occured while fetching bills list');
-                return next('UNKNOWN_ERROR');
+                return next('LOGIN_FAILED');
```

Comment:
That's incorrect, the login succeeded but something failed after that (ex: the bills URL has changed).